firefox_decrypt.py

```diff
@@ -1110,12 +1110,14 @@ def main() -> None:
     # Finally shutdown NSS
     moz.unload_profile()
 
-
-if __name__ == "__main__":
+def run():
     try:
         main()
     except KeyboardInterrupt:
         print("Quit.")
         sys.exit(Exit.KEYBOARD_INTERRUPT)
     except Exit as e:
         sys.exit(e.exitcode)
+
+if __name__ == "__main__":
+    run()
```

Comment:
Can you add a newline at the end of this file? Otherwise it will be added in a future commit.
Thanks!